governance/executive-council.md

```diff
@@ -0,0 +1,261 @@
+
+:::{note}
+While travel is not explicitly required, members may have opportunities to attend meetings on behalf of pyOpenSci. Travel may be supported by pyOpenSci if needed.
+:::
+
+
+## How the Executive Council makes decisions
+
+The Executive Council is required to review and approve decisions affecting the fiduciary aspects of the organization; this includes mission/vision/values, governance policies, budget, Executive Director compensation, and significant new strategic initiatives. The Executive Council also supports the Executive Director by serving as a resource for input when needed.
+
+All decisions made by the Executive Council must align with the legal
+obligations and policies of pyOpenSci’s fiscal sponsor—[Community Initiatives](https://communityinitiatives.org/).
+This ensures compliance with overarching governance structures and financial
+responsibilities.
+
+
+### Decisions requiring an Executive Council vote
+
+The Executive Council (EC) defines and directs pyOpenSci’s strategy and major
+organizational direction. The EC approves significant decisions
+that strongly impact pyOpenSci’s strategy, governance, or financial well-being.
+Generally, items with legal or fiduciary implications require a formal vote,
+with the outcome documented in this handbook's Executive Council [meeting minutes](ec-minutes) section.
+
+The EC will vote on the following:
+
+- **Annual budget**: Approval of the organization's yearly budget.
+- **Mission, vision, and values**: Decisions related to pyOpenSci’s core principles.
+- **Governance and policy changes**: Approval of changes to governance structures,
+  including updates to the Code of Conduct.
+- **Significant budget deviations**: Decisions involving major deviations from the
+  approved annual budget.
+- **Financial health implications**: Any decision with substantial implications for
+  pyOpenSci’s financial stability.
+- **Strategic direction**: Approval of major changes to organizational strategy and
+  development roadmaps.
+- **Executive director’s salary**: The EC independently determines the salary of the
+  Executive Director.
+
+:::note
+The Executive Director is authorized to make all other decisions. However, the
+Executive Council serves as a resource for the Executive Director and may provide
+feedback and input upon request.
+:::
+
+### Decision-making process
+
+The topics above require a motion and a vote for approval. Motions and approval can be made in Executive Council meetings or asynchronously via email or Slack. Motions and votes will be noted in the EC meeting minutes.
+
+:::{important}
+All decisions made by the Executive Council must align with the legal
+obligations and policies of pyOpenSci’s fiscal sponsor—[Community Initiatives](https://communityin.org/).
+:::
+
+## Collaboration between the Executive Director and the Executive Council
```

Comment:
Move this section into the "operations at the bottom too" . 

and remove the staff lines from the document. (we can use may request feedback - in the other operations section).

Reply:
Moved this and lazy consensus to the operations section.